src/RadioButton/RadioButton.js

```diff
@@ -91,12 +91,18 @@ const RadioButton = React.createClass({
     labelPosition: React.PropTypes.oneOf(['left', 'right']),
 
     /**
-     * Overrides the inline-styles of the RadioButton element label.
+     * Override the inline-styles of the label element.
      */
     labelStyle: React.PropTypes.object,
 
     /**
-     * Callback function for checked event.
+     * Callback function fired when the radio button is checked. Note that this
+     * function will not be called if the radio button is part of a
+     * radio button group: in this case, use the `onChange` property of
+     * `RadioButtonGroup`.
+     *
+     * @param {object} event `change` event targeting the element.
+     * @param {string} value The element's `value`.
      */
     onCheck: React.PropTypes.func,
```

Comment:
I can't think of any use for it outside of a `RadioButtonGroup`; if `checked` was a public prop on `RadioButton` it could be used independently, but since it isn't `@ignore` seems better.